ingress/controllers/nginx/Makefile

```diff
@@ -4,8 +4,16 @@ all: push
 TAG = 0.5
 PREFIX = gcr.io/google_containers/nginx-ingress-controller
 
+REPO_INFO=$(shell git config --get remote.origin.url)
```

Comment:
@bprashanth why? that code is just add additional information about the build from git. No user intervention required. This is the first line in the logs:
```
I0410 21:36:01.457316       1 main.go:89] Using build: https://github.com/aledbf/contrib - git-b580f13
```